virtual_rainforest/core/model.py

```diff
@@ -99,3 +103,41 @@ def __repr__(self) -> str:
     def __str__(self) -> str:
         """Inform user what the model type is."""
         return f"A {self.name} model instance"
+
+
+class FailedModel(BaseModel, model_name="failed"):
```

Comment:
I really don't see the point of this Failed model. If a model fails to be constructed, then an execution should be raised. If we want o collect these exception and let the show go on with the rest of the models until we have constructed all of them or just stop things there is a separate question, as we discussed the other day. But this model, in itself just a placeholder that does nothing, does not seem to me as the right way to go.